src/pages/activities/edit/[id].vue

```diff
@@ -44,12 +45,25 @@ function onLeaveActivityWeekForm() {
 onBeforeMount(async () => {
   await useActivityWeekStore().api.query({ activity_id: id })
 })
+
+const items = [
+  {
+    text: 'Actividades',
+    color: 'primary',
+    disabled: false,
+    to: '/activities',
+  },
+  {
+    text: 'Editar Actividad',
+    disabled: true,
+    to: '/activities',
```

Comment:
El to: no debería ser necesario en el último elemento de los ítems, sugiero borrar en todos los de más items

Reply:
Se cambio a to:'#'